File: app/src/main/java/com/learning/helloworld/ChildActivity.java
package com.learning.helloworld;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

public class ChildActivity extends AppCompatActivity {

    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_child);
    }
}

File: app/src/main/java/com/learning/helloworld/ChildActivity.kt
package com.learning.helloworld

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class ChildActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_child)
    }
}
